contributing/documentation/overview.rst

```diff
@@ -107,6 +107,12 @@ your project:
     upstream  https://github.com/symfony/symfony-docs.git (fetch)
     upstream  https://github.com/symfony/symfony-docs.git (push)
 
+Fetch the upstream branch list.
```

Comment:
In Step 4 we use directly the `upstream/` to create the branch:

```
$ git checkout -b improve_install_chapter upstream/2.7
```

My question: is really mandatory to execute `git fetch upstream` first?

---

In case it's mandatory, I propose to reword `Fetch the upstream branch list.` a bit to explain what's happening. For example:

```
Fetch all the commits of the upstream branches executing
this command:
```